web-jehlomat/src/config/baseURL.ts

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 const HOST = process.env.REACT_APP_SERVER ?? "localhost";
 const PORT = process.env.REACT_APP_SERVER_PORT ?? 8082;
-const BASE_URL = `${HOST}:${PORT}`;
+const BASE_URL = `http://${HOST}:${PORT}`;
```

Comment:
This is breaking change for people who has .env file already setup this way. Please notify everybody in slack for example :)

Reply:
or update the logic, so it adds port only if its missing in original url